orgs/opentelekomcloud-docs/repositories/map-reduce-service.yml

```diff
@@ -0,0 +1,19 @@
+---
+map-reduce-service:
```

Comment:
shouldn't we use "mapreduce-service" instead of "map-reduce-service" 
similarly we used "modelarts" and not "model-arts"